crates/config/src/config.rs

```diff
@@ -302,6 +303,12 @@ impl Default for PruneConfig {
     }
 }
 
+impl From<PruneConfig> for PrunerBuilder {
+    fn from(value: PruneConfig) -> Self {
+        PrunerBuilder::default().block_interval(value.block_interval).segments(value.segments)
+    }
+}
```

Comment:
the config is only useful for the Builder,
can we reverse the imports and move this to `PrunerBuilder::new(Config)`

I also want to do the same for

https://github.com/paradigmxyz/reth/blob/b1d0dc41c2381951c0153a20dc1381b2434bca2a/crates/config/src/config.rs#L178-L178

Reply:
right, we use this config->builder pattern a lot, so importing the configs for builder `new` methods makes sense.

created an issue for bodies config https://github.com/paradigmxyz/reth/issues/5835